sygnal/apnspushkin.py

```diff
@@ -168,12 +168,12 @@ def __init__(self, name: str, sygnal: "Sygnal", config: Dict[str, Any]) -> None:
         # use the Sygnal global proxy configuration
         proxy_url_str = sygnal.config.get("proxy")
 
-        loop = asyncio.get_event_loop()
+        loop = asyncio.get_running_loop()
         if proxy_url_str:
             # this overrides the create_connection method to use a HTTP proxy
             loop = ProxyingEventLoopWrapper(loop, proxy_url_str)  # type: ignore
 
-        async def make_apns() -> aioapns.APNs:
+        def make_apns() -> aioapns.APNs:
```

Comment:
I'd be tempted to remove the function and just set `self.apns_client` directly, i.e. instead of `return` just `self.apns_client = `